flowsa/methods/flowbysectormethods/GHG_national_m1.yaml

```diff
@@ -0,0 +1,710 @@
+    helper_method: proportional
+    helper_source_class: "Money"
+    helper_source_year: 2012
+    helper_flow: ["USD2012"]
+    helper_from_scale: national
+    clean_helper_fba_wsec: !script_function:BEA subset_and_equally_allocate_BEA_table
+    clean_parameter: {"324110": 'ActivityProducedBy'} # use of petroleum products
+
+source_names:
+  "EPA_GHGI_T_2_1": #U.S. GHG emissions
+    data_format: 'FBA'
+    class: Chemicals
+    geoscale_to_use: national
+    # year: # override this
+    fedefl_mapping: 'GHGI'
+    activity_sets:
+      direct:
+        names:
+          - "Iron and Steel Production & Metallurgical Coke Production" #CO2, CH4
+          - "Cement Production" #CO2
+          - "Lime Production" #CO2
+          - "Ammonia Production" #CO2
+          - "Incineration of Waste" #CO2, #N2O
+          - "Aluminum Production" #CO2
+          - "Soda Ash Production" #CO2
+          - "Ferroalloy Production" #CO2, CH4
+          - "Titanium Dioxide Production" #CO2
+          - "Zinc Production" #CO2
+          - "Phosphoric Acid Production" #CO2
+          - "Glass Production" #CO2
+          - "Carbide Production and Consumption" #CO2, CH4
+          - "Landfills" #CH4
+          - "Coal Mining" #CO2, CH4
+          - "Wastewater Treatment" #CH4, N2O
+          - "Rice Cultivation" #CH4
+          - "Abandoned Oil and Gas Wells" #CH4
+          - "Abandoned Underground Coal Mines" #CH4
+          - "Anaerobic Digestion at Biogas Facilities" #CH4 new activity
+          - "Composting" #CH4, N2O
+          - "Nitric Acid Production" #N2O
+          - "Adipic Acid Production" #N2O
+          - "Caprolactam, Glyoxal, and Glyoxylic Acid Production" #N2O
+        source_flows: ["CO2", "CH4", "N2O"] # HFCs and other flows are pulled elsewhere
+        allocation_method: direct
+        allocation_source: None
+        allocation_from_scale: national
+      electricity_transmission:
+        names:
+          - "Electrical Transmission and Distribution" #SF6
+        source_flows: ["SF6"]
+        allocation_method: direct
+        allocation_source: None
+        allocation_from_scale: national
+      electric_power:
+        <<: *make_allocation
+        names:
+        - "Electric Power Sector" #CO2
+        clean_parameter: {"221100": 'ActivityConsumedBy'} # Electricity
+      liming:
+        <<: *use_allocation
+        names:
+        - "Liming"  #CO2
+        clean_parameter: {"327400": 'ActivityProducedBy'} # Lime
+      urea:
+        <<: *use_allocation
+        names:
+        - "Urea Fertilization" #CO2
+        - "Urea Consumption for Non-Agricultural Purposes" #CO2
+        clean_parameter: {"325310": 'ActivityProducedBy'} # Fertilizers
+      carbonates:
+        <<: *use_allocation
+        names:
+        - "Other Process Uses of Carbonates" #CO2
+        clean_parameter: {"325180": 'ActivityProducedBy'} # Other Basic Inorganic Chemicals
+      lead:
+        <<: *lead_allocation
```

Comment:
@catherinebirney or @bl-young , could you confirm "Lead Production" in Table 2-1 is not used here?

Reply:
no, it is used, see line 71 which is where the name is defined (via Table 2-1)

I think the way it's structured currently in the yaml is not ideal and so I will likely move lines 70-71 down into this section here.

Reply:
I see, thanks Ben!

I want to completely decompose this config file to be able to generate FBS for each source_name after line 126, e.g. Table 2-1 Rice Cultivation, Table 2-1 Coal Mining, etc. I'm planning to go through every `name` after line 126, comment (toggle off) all other names, and save a yaml for that name only. But since a few names (lead, cropland_allocation, etc) are reusing definitions before line 126, should I also toggle off the definition if I want it excluded those names?

Reply:
does d6b8561 help?

Yes if you want to do each individual activity by individually I think that would be what you'd have to do.

Reply:
Yes, very helpful. Thanks Ben!